docs/tutorial/notifications.md

```diff
@@ -5,7 +5,9 @@ to the user. Electron conveniently allows developers to send notifications with
 the [HTML5 Notification API](https://notifications.spec.whatwg.org/), using
 the currently running operating system's native notification APIs to display it.
 
-**Note:** Since this is an HTML5 API it is only available in the renderer process.
+**Note:** Since this is an HTML5 API it is only available in the renderer process. If
+you want to show Notifications in the main process please check out the
```

Comment:
it is not really "show in main process", but more like "Invoke notification from main process" IMO, just my $0.02 😄